rtgui/indclippedpanel.cc

```diff
@@ -34,39 +34,39 @@ IndicateClippedPanel::IndicateClippedPanel (ImageArea* ia) : imageArea(ia)
 
     Glib::ustring tt;
 
-    indclippedh = Gtk::manage (new Gtk::ToggleButton ());
-    indclippedh->set_relief(Gtk::RELIEF_NONE);
-    indclippedh->add (*Gtk::manage (new RTImage ("warnhl.png")));
-    tt = Glib::ustring::compose("%1\n%2 = %3", M("MAIN_TOOLTIP_INDCLIPPEDH"), M("MAIN_TOOLTIP_THRESHOLD"), options.highlightThreshold);
+    indClippedH = Gtk::manage (new Gtk::ToggleButton ());
+    indClippedH->set_relief(Gtk::RELIEF_NONE);
+    indClippedH->add (*Gtk::manage (new RTImage ("warnhl.png")));
+    tt = Glib::ustring::compose("%1\n%2 = %3", M("MAIN_TOOLTIP_indClippedH"), M("MAIN_TOOLTIP_THRESHOLD"), options.highlightThreshold);
```

Comment:
Language strings should remain all uppercase.

Reply:
Yeah, that was unintended.